### PR TITLE
Added enumeration descriptions for the VM Connector Reference

### PR DESCRIPTION
**Pull request by dhirsch1**
Fixed DOCS-4294 by adding descriptions to the enumerations for the Transactional Action parameter.

### CODE REVIEW

File: modules/ROOT/pages/vm/vm-reference.adoc [hunk 1]
```diff
@@ -5,9 +5,9 @@ endif::[]
...
 
-When running on a single instance, persistent queues work by serializing and storing the contents on to disk. When running in cluster mode, persistent queues are instead backed by the memory grid. This means that when a flow uses the VM connector to publish content to a queue, the Runtime decides whether to process that message in the same origin node or to send it out to the cluster for another node to pick it up. This is an easy way to distribute load across the cluster. In either way, transactions are always supported. 
+When running on a single instance, persistent queues work by serializing and storing the contents on to disk. When running in cluster mode, persistent queues are instead backed by the memory grid. This means that when a flow uses the VM connector to publish content to a queue, the Runtime decides whether to process that message in the same origin node or to send it out to the cluster for another node to pick it up. This is an easy way to distribute load across the cluster. In either way, transactions are always supported.
```
Comment (valkyrie69) [inline modules/ROOT/pages/vm/vm-reference.adoc:10]: Can't use "Runtime" Have to use Mule or Mule runtime engine

Last line should be loads (I think?)
I think the last sentence is totally unnecessary.

File: modules/ROOT/pages/vm/vm-reference.adoc [hunk 2]
```diff
@@ -158,9 +160,9 @@ Publishes the given content into the queue of the given queueName.
 
 `<vm:publish-consume>`
 
-Publishes the given content into a queue, and then waits up to the queueTimeout for a response to be supplied on a temporal reply-To queue that this operation automatically creates. 
+Publishes the given content into a queue, and then waits up to the queueTimeout for a response to be supplied on a temporal reply-To queue that this operation automatically creates.
```
Comment (valkyrie69) [inline modules/ROOT/pages/vm/vm-reference.adoc:163]: Is temporal right, or is it temporary?

File: modules/ROOT/pages/vm/vm-reference.adoc [hunk 3]
```diff
@@ -158,9 +160,9 @@ Publishes the given content into the queue of the given queueName.
...
 
-The temporal reply queue is automatically disposed after a response is received or the timeout expires. 
+The temporal reply queue is automatically disposed after a response is received or the timeout expires.
```
Comment (valkyrie69) [inline modules/ROOT/pages/vm/vm-reference.adoc:165]: temporal or temporary?
```suggestion
The temporary reply-to queue is deleted automatically after a response is received or when the timeout expires.
```

File: modules/ROOT/pages/vm/vm-reference.adoc [hunk 4]
```diff
@@ -170,12 +172,14 @@ The queue in which the content is published cannot contain a Listener (`<vm:list
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| Content a| Any |  +++the content to be published+++ |  +++#[payload]+++ | {nbsp}
+| Content a| Any |  +++The content to be published+++ |  +++#[payload]+++ | {nbsp}
```
Comment (valkyrie69) [inline modules/ROOT/pages/vm/vm-reference.adoc:175]: Did this get run through the cleanup script? That usually deletes all occurrences of {nbsp}

**Review by valkyrie69 — APPROVED** [review]
Minor comments. Did this get run through the cleanup script?

**Comment by dhirsch1** [comment]
Took notes of review comments for other parts of the doc so I can implement them when the Reference Guide version is updated.

**Pull request merged**